src/WindowsAzureMessaging/WindowsAzureMessaging/MSNotificationHub.m

```diff
@@ -64,6 +64,11 @@ - (void)registerForRemoteNotifications {
 - (BOOL)didReceiveRemoteNotification:(NSDictionary *)userInfo {
     MSNotificationHubMessage *message = [MSNotificationHubMessage createFromNotification:userInfo];
     [self didReceivePushNotification:message];
+    
+    if(message.additionalData)
```

Comment:
Nit: you can use ternary operator ?: here

Reply:
👍